app/controllers/submissions_controller.rb

```diff
@@ -46,8 +46,9 @@ def create
     @submission.referer = submission_params[:referer]
     @submission.page = submission_params[:page]
     @submission.ip_address = request.remote_ip
-
-    create_in_local_database(@submission)
+    @result = create_in_local_database(@submission)
+    @form.update(response_count: @form.submissions.size, last_response_created_at: @submission.created_at)
```

Comment:
how about doing this in a callback on `Form.rb` - `before_update`

Reply:
Check out `counter_cache` https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to